src/Sulu/Bundle/WebsiteBundle/Routing/PortalRoute.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\WebsiteBundle\Routing;
+
+use Symfony\Component\Routing\Route;
+
+/**
+ * Extends symfony route to avoid // in the url.
+ */
+class PortalRoute extends Route
+{
+    /**
+     * Trims leading slash to avoid "//".
+     *
+     * @return mixed
+     */
+    public function getPath()
+    {
+        return ltrim(parent::getPath(), '/');
```

Comment:
Are you sure you are not passing something wrong somewhere else? Seems like an overkill to create an own class for this purpose.